src/test/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/EventSpec.scala

```diff
@@ -20,6 +20,7 @@ import java.util.UUID
 // cats
 import cats.data.Validated.{Invalid, Valid}
 import cats.data.NonEmptyList
+import cats.syntax.either._
```

Comment:
Why do we need this?

Reply:
It is in this [commit](https://github.com/snowplow/snowplow-scala-analytics-sdk/pull/79/commits/b1afe5d7ebeabd50c86cc9b2ab62ab468c9575de). Should I remove it ?

Reply:
Yeah, I suppose so.